NewsApp/NewsApp/Model/Services/NewsService.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  NewsService.swift
+//  NewsApp
+//
+//  Created by Josselin DUBOC on 30/03/2020.
+//  Copyright © 2020 Josselin DUBOC. All rights reserved.
+//
+
+import Foundation
+
+protocol NewsService {
+    var newsArticles: [Article] { get set }
```

Comment:
est-ce que c'est utile ici d'avoir newsArticles en set ?

On consomme le NewsService pour récupérer la liste des news articles mais jamais de besoin de changer la valeur ?

Reply:
fix in "FixPR"